File: Iniciante/1073.c
#include <stdio.h>

int main() {

    int N, i;

    scanf("%d", &N);

    for(i = N - (N-1); i <= N; i++){
        if(i % 2 == 0) printf("%d^2 = %d\n", i, i*i);
    }    

    return 0;
}

File: Iniciante/1073.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);
        
        int N = scanf.nextInt();

        for(int i = N - (N-1); i <= N; i++){
        if(i % 2 == 0) System.out.printf("%d^2 = %d\n", i, i*i);
        }    
    }
}